src/docs/sphinx/blueprint_mesh.rst

```diff
@@ -9,10 +9,19 @@ Mesh Blueprint
 ===================
 
 This section provides details about the Mesh Blueprint. Lots of them.
-We don't have a Mesh Blueprint tutorial yet, if you are looking to wrap your mind 
-around the basic mechanics of describing a mesh, you may want to start by reviewing
-the :ref:`detailed_uniform_example`
-and exploring the other :ref:`examples` included in the blueprint library. 
+These docs provide the main reference for all of the components of
+the Mesh Blueprint protocol and details about :ref:`examples`
+that are included in the Conduit Blueprint Library. 
+
+
+Conduit docs don't have a Mesh Blueprint tutorial yet, if you are looking to 
+wrap your mind around the basic mechanics of describing a mesh:
+
+* The Ascent tutorial includes section on `creating Meshes using Conduit <https://ascent.readthedocs.io/en/latest/Tutorial_Intro_Conduit_Blueprint.html>`_. This is the best reference for getting started and includes C++ and Python code examples.
+
+* The :ref:`complete_uniform_example` at the end of this section shows you how to create and save a uniform grid to a file which VisIt and `Ascent's Replay utility <https://ascent.readthedocs.io/en/latest/Utilities.html?highlight=replay#replay>`_ can read.
+
+* The :ref:`examples` section details functions that to generate several flavors of exemplar meshes.
```

Comment:
These are great resources; thanks for including them, and thanks to you and/or @mclarsen for writing them!